Makefile

```diff
@@ -17,6 +17,9 @@ COORDINATOR_ARGS=-mod=readonly -trimpath ./cmd/ltcoordinator
 AGENT=ltagent
 AGENT_ARGS=-mod=readonly -trimpath ./cmd/ltagent
 
+export GOBIN = $(PWD)/bin
```

Comment:
I somehow prefer the var assignment without spaces, mainly because I've been hit by this a few times in the past.